src/core/MOM_PressureForce_FV.F90

```diff
@@ -1227,7 +1242,7 @@ subroutine PressureForce_FV_Bouss(h, tv, PFu, PFv, G, GV, US, CS, ALE_CSp, p_atm
     if (CS%sal_use_bpa) then
       !$OMP parallel do default(shared)
       do j=Jsq,Jeq+1 ; do i=Isq,Ieq+1
-        pbot(i,j) = pa(i,j,nz+1) - GxRho * e(i,j,nz+1)
+        pbot(i,j) = pa(i,j,nz+1) - GxRho_ref * e(i,j,nz+1)
```

Comment:
I agree with the change on this line from `GxRho` to `GxRho_ref`, however I suspect there is a separate bug with a missing reference geopotential height, and that this line should read `pbot(i,j) = pa(i,j,nz+1) - GxRho_ref * (e(i,j,nz+1) - G%Z_ref)`.

Reply:
Thanks, good catch!